spec/acceptance/foreman_journald_spec.rb

```diff
@@ -91,10 +91,10 @@ class { '::foreman':
   end
 
   describe command("journalctl -u #{service_name}") do
-    its(:stdout) { is_expected.to match(%r{Redirected to https://#{host_inventory['fqdn']}/users/login}) }
+    its(:stdout) { is_expected.to match(%r{Started The Apache HTTP Server}) }
```

Comment:
This test was intended to test that the application logs to the journal. That means what's normally in production.log is now in the journal. By changing this to only test the regular messages, it's missing the point. Why that's not actually happening is unknown to me and may be an indication of an actual bug.

Reply:
I revereted these and they passed for me locally. I don't know what I was seeing.